linters/kubebench/subscriber.go

```diff
@@ -156,7 +156,7 @@ func (s *Subscriber) findNodesForScan() []*nodeJob {
 	nodes := s.delta.peek()
 	var res []*nodeJob
 	for _, nodeJob := range nodes {
-		if nodeJob.ready() {
+		if nodeJob.ready() && len(nodeJob.node.Spec.Taints) == 0 {
```

Comment:
Also here https://github.com/castai/kvisor/blob/main/linters/kubebench/subscriber.go#L204
lets add waitJobDeleted() which checks that job is actually removed before we schedule new one